modules/vardictjava/main.nf

```diff
@@ -34,7 +34,7 @@ process VARDICTJAVA {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        vardictjava: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//' ))
+        vardictjava: \$(echo 1.8.3)
```

Comment:
This should be a command that fetcheds the version number, not a hardcoded version number

Reply:
VarDict has no option to fetch the version number command-wise